### PR TITLE
docs(website): add recent blogs on home page

### LINKED ISSUE #800: Add recent blog posts section to website front page
**Is your feature request related to a problem? Please describe.**
We just redesigned our front page of [glasskube.dev](https://glasskube.dev/) and publish [blog posts](https://glasskube.dev/blog/) regularly, but the latest posts are not visible on the website.

**Describe the solution you'd like**
Add a recent blog post section to the front-page, similar to the [cyclops](https://cyclops-ui.com/) website:

image

**Additional context**
The blog posts should be fetched dynamically.

### PR DESCRIPTION
**Pull request by Aryakoste**
Closes #800 

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

File: website/src/components/HomepageBlogs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          {recentPosts.map((post, index) => (
+            <div key={index} className={clsx('card', styles.blogCard)}>
+                <div className={clsx('card__image', styles.blogCardImage)}>
+                  <img src={post.metadata.frontMatter.image}/>
+                </div>
+                <div className={clsx(styles.cardContent, 'card__body')}>
+                <img src={post.Preview.metadata.authors[0].imageURL} alt={post.Preview.metadata.authors[0].name} className={styles.authorImage} />
+                <div className={styles.cardDesc}>
+                  <h3><Link to={post.Preview.metadata.permalink}>{post.metadata.title}</Link></h3>
+                  <p>{post.metadata.description}</p>
```
Comment (kosmoz) [inline website/src/components/HomepageBlogs/index.tsx:49]:
```suggestion
                  <div>{post.metadata.description}</div>
```

**Review by kosmoz — CHANGES_REQUESTED** [review]
I think this looks really good for a start. Please check my comments below regarding the styling.

Additionally, the build seems to be failing because home.tsx is interpreted to be a page because it is in the pages directory. You can fix that my moving home.tsx and index.module.css to /src/components/home/ and fixing the reference in the custom blog plugin.

Lastly, please make sure to format your newly created files using `npx prettier -w src/components/HomepageBlogs/*`.

**Comment by Aryakoste** [comment]
@kosmoz Changes done.

**Review by kosmoz — APPROVED** [review]

File: website/src/components/HomepageBlogs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  return (
+    <section className={styles.latestBlogPosts}>
+      <div className="container margin-top--lg">
+        <div className="row">
+          <div className="col text--center">
+            <h2>Check our blog</h2>
+            <p>
+              We are working on a blog post series covering topics from general
+              open source to specific Kubernetes related topics.{' '}
+              <Link to={useBaseUrl('/blog')}>Check it out!</Link>
```
Comment (pmig) [inline website/src/components/HomepageBlogs/index.tsx:23]:
```suggestion
              We are regularly writing <Link to={useBaseUrl('/blog')}>blog posts</Link> and publish <Link to={useBaseUrl('//guides/categories/')}>guides</Link>. Find our latest blog posts below.
              
```

**Review by pmig — CHANGES_REQUESTED** [review]
Thanks for you contribution I merged another pull request removing the the ASCII video. Could you fix the merge conflicts?

I also requests some smaller (wording) changes.

**Comment by Aryakoste** [comment]
@pmig @kosmoz Changes done.

**Review by pmig — APPROVED** [review]

**Pull request merged**